pandas/core/groupby/generic.py

```diff
@@ -914,6 +914,25 @@ def fillna(
         --------
         ffill : Forward fill values within a group.
         bfill : Backward fill values within a group.
+
+        Examples
+        --------
+        For SeriesGroupBy:
+
+        >>> lst = ['a', 'a', 'b', 'b']
+        >>> ser = pd.Series([1, np.nan, 3, np.nan], index=lst)
+        >>> ser
+        a    1.0
+        a    NaN
+        b    3.0
+        b    NaN
+        dtype: float64
+        >>> ser.groupby(level=0).fillna(value=0)
```

Comment:
this one works, but isn't how one would usually use `groupby.fillna` - for this example, you could just do `ser.fillna(0)`

how about
```python
ser.groupby(level=0).fillna(method='ffill')
```
to better show what's unique about this method?

Reply:
I thought that the keyword `method` there was deprecated https://github.com/pandas-dev/pandas/pull/53496

Reply:
ah you're right, thanks!